package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "libertybans",
+  "version": "1.0.0",
```

Comment:
I would prefer not to have to update this version each time there is a release - it would be easy to miss. Is there a way to avoid specifying the version here?